hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/utils/FileServiceUtils.java

```diff
@@ -81,6 +84,11 @@ public static void validateContent(
 
         final var fileMeta = fileStore.getFileMetadata(fileId);
         mustExist(fileMeta, INVALID_FILE_ID);
+
+        if (isSpecialFile && fileId.fileNum() > 750) {
```

Comment:
Make certain to replace the constant here with config (You'll need to pass in context or the value read from config) before merging.  This value has changed in the past and likely will change again in the future.

Reply:
I changed it to read it from the Ledger config file.